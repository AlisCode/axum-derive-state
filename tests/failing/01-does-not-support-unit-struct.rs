//! Only structures and tuple structures are supported.
//! Unit structs are not supported.

use derive_state::State;

#[derive(State)]
pub struct MyState;

fn main() {}
