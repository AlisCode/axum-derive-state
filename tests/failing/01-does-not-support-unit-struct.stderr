error: The state of your app may never be empty
 --> tests/failing/01-does-not-support-unit-struct.rs:7:1
  |
7 | pub struct MyState;
  | ^^^^^^^^^^^^^^^^^^^
